### ISSUE TITLE
When we create a new redaction , all other redactions get the time stamp same as the newly created ones(After a refresh)

### ISSUE THREAD

**Issue by nimya-aot** [bcgov/foi-flow#4603]
**Describe the bug in current situation**
When we create a new redaction, all other redactions get time stamped as  the new one after a refresh(Check the time stamp on 'Marked for Redactions') 

**Link bug to the User Story**

**Impact of this bug**
Describe the impact, i.e. what the impact is, and number of users impacted. 

**Chance of Occurring (high/medium/low/very low)**
Medium

**Pre Conditions: which Env, any pre-requesites or assumptions to execute steps?**
test-marshal

**Steps to Reproduce**
1. Go to redaction app
2. Try to create a redaction on any page 
3. Check the 'Marker for redaction'  section and it appears there
4. Do a refresh and can see all other redactions get stamped with the newly created redaction's time stamp

**Actual/ observed behaviour/ results**

**Expected behaviour**
A clear and concise description of what you expected to happen. Use the gherking language. 

**Screenshots/ Visual Reference/ Source**
If applicable, add screenshots to help explain your problem. You an use screengrab.

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBencvQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f60ed0b5131b5cb0de5faf7ec0650d4d43fc3c4b/image.png)

[Uploading... image.png]

**Comment by lmullane** [comment]
@m-prodan, @liseandtea, this seems to me like a problem. Analysts and peer reviewers would likely want to know when a redaction was made, correct?

**Comment by m-prodan** [comment]
I don't think this is a show-stopper, as far as I know, we don't know when redactions were made in AXIS... at least not from the front-end as a user. But definitely something we should fix for audit purposes in the future. I don't think this should be required for 9.0 at least.

**Comment by liseandtea** [comment]
@lmullane, @m-prodan I agree that this is not a complete show stopper. I only think this is a problem if it changes the order of the records in any way. As a peer reviewer, I have never checked to see exactly when a redaction was done. I just look to see that a box is there.

**Comment by KyEggleston** [comment]
Confirmed still occurring.
